egs/m_ailabs/tts1/conf/train_pytorch_tacotron2.yaml

```diff
@@ -0,0 +1,46 @@
+# This is the basic tactron2 training settting
+
+# encoder related
+embed-dim: 512
+elayers: 1
+eunits: 512
+econv-layers: 3 # if set 0, no conv layer is used
+econv-chans: 512
+econv-filts: 5
+
+# decoder related
+dlayers: 2
+dunits: 1024
+prenet-layers: 2  # if set 0, no prenet is used
+prenet-units: 256
+postnet-layers: 5 # if set 0, no postnet is used
+postnet-chans: 512
+postnet-filts: 5
+
+# attention related
+atype: location
+adim: 128
+aconv-chans: 32
+aconv-filts: 15      # resulting in filter-size = aconv-filts * 2 + 1
+cumulate-att-w: true # whether to cumulate attetion weight
+use-batch-norm: true # whether to use batch normalization in conv layer
+use-concate: true    # whether to concatenate encoder embedding with decoder lstm outputs
+use-residual: false  # whether to use residual connection in encoder convolution
+use-masking: true    # whether to mask the padded part in loss calculation
+bce-pos-weight: 1.0  # weight for positive samples of stop token in cross-entropy calculation
+reduction-factor: 2
```

Comment:
Check the values of `atype` and `reduction-factor`.

Reply:
Same as the above.